.evergreen/config/test/_template-load-balanced.yml

```diff
@@ -0,0 +1,15 @@
+tasks:
+  - name: "test-mongodb-%mongodbVersion%-loadbalanced"
+    tags: ["sharded", "local", "%mongodbVersion%", "loadbalanced"]
+    commands:
+      - func: "bootstrap mongo-orchestration"
+        vars:
+          TOPOLOGY: "sharded_cluster"
+          MONGODB_VERSION: "%mongodbVersion%"
+          LOAD_BALANCER: "true"
+          SSL: "yes"
+      - func: "start load balancer"
+      - func: "run tests"
+        vars:
+          MONGODB_URI: "${SINGLE_MONGOS_LB_URI}"
+          SSL: "yes"
```

Comment:
Side note: I have long disliked how `SSL` uses "yes|no" and `AUTH` uses "auth|noauth"; however, I this is is largely influenced by [run-orchestration.sh](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/run-orchestration.sh#L5), so changing it for our own `run-tests.sh` script would just invite inconsistency without our own Evergreen configs.